lib/elements/autocomplete.js

```diff
@@ -140,13 +149,29 @@ class AutocompletePrompt extends Prompt {
   }
 
   down() {
-    if (this.select >= this.suggestions.length - 1) return this.bell();
+    if (this.select >= this.suggestions[this.page].length - 1) return this.bell();
     this.moveSelect(this.select + 1);
     this.render();
   }
 
   next() {
-    this.moveSelect((this.select + 1) % this.suggestions.length);
+    this.moveSelect((this.select + 1) % this.suggestions[this.page].length);
```

Comment:
As mentioned in the conversation, it would be nice to jump to the next page if we hit the last element.

Reply:
Yeah that's a good idea

Reply:
Done!

Reply:
I'm testing this on latest master ( e841383 ) and am not experiencing the auto paging when I get to the last result. I posted some comments below